public/js/main.js

```diff
@@ -67,4 +67,85 @@ $(document).ready(function () {
         e.preventDefault();
         window.location = "/api/csv/1";
     });
+
+    /**
+     * Functions for User List
+     */
+
+    $(".make_admin").click(function (e) {
```

Comment:
Can we split these out into a separate js file? I know we're still being janky with jquery, but we should keep it modular to some extent.